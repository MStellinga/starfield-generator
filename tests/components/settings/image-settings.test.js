/* eslint-disable jest/valid-expect */
import { h } from 'preact';
import render from 'preact-render-to-string';
import ImageSettings from '../../../src/components/settings/image-settings';

describe('Test Image settings', () => {
	test('Image settings renders', () => {
        const tree = render(<ImageSettings width={1} height={2} />);
        expect(tree).toMatchSnapshot();	
    });

    // Unfortunately, adding tests that call event handlers is hard to do with Enzyme and function-based Preact components (at this time, anyway)
});
