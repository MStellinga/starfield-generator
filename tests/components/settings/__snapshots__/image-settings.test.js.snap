// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Image settings Image settings renders 1`] = `"<div class=\\"section\\"><div>Image size <input type=\\"text\\" pattern=\\"-?[0-9\\\\.]{1,5}\\" class=\\"smallfield\\" value=\\"1\\" /> x <input type=\\"text\\" pattern=\\"-?[0-9\\\\.]{1,5}\\" class=\\"smallfield\\" value=\\"2\\" /></div></div>"`;
